### PR TITLE
Quality update 7

### PR DESCRIPTION
**Pull request by x9136**
I round corners, simplify shapes, correct sizes, and change the fill to stroke. 

Сhanged the Phone Master icon, because the one linked before didn't fit.

## New icons added
Phone Master (`com.transsion.phonemaster/com.cyin.himgr.ads.SplashActivity`)
ReadEra Premium (`org.readera.premium/com.readera.MainActivity`)
## Updated due to redesign
App vault (`com.mi.android.globalminusscreen`)
Microsoft Office (`com.microsoft.office.officehubrow`)
Reverso Context (`com.softissimo.reverso.context`)
## Linked
`inpost.svg` to (`pl.inpost.inmobile/pl.inpost.inmobile.splashScreen.presentation.SplashScreenActivity`)
`fruit_ninja.svg` to (`com.halfbrick.fruitninja/com.halfbrick.mortar.MortarGameActivity`)
`fruit_ninja.svg` to (`com.halfbrick.fruitninja/com.halfbrick.mortar.MortarGameLauncherActivity`)
`spotify.svg` to (`com.spotify.musix/com.spotify.musix.MainActivity`)
`translate_you.svg` to (`com.bnyro.translate.nf/com.bnyro.translate.ui.MainActivity`)

### CODE REVIEW

File: svgs/edupage.svg
Comment (KTibow) [inline svgs/edupage.svg:1]: Please use a `<rect>` instead of drawing a square manually

Comment (x9136) [reply svgs/edupage.svg:1]: The plugin I use for optimization removes the `<rect>`. Would you be satisfied with this code?

Before
```
<svg width="192" height="192" viewBox="0 0 192 192" fill="none" xmlns="http://www.w3.org/2000/svg">
<rect x="30" y="30" width="132" height="132" rx="6" stroke="black" stroke-width="12" stroke-linecap="round" stroke-linejoin="round"/><path d="M58 96H124" stroke="black" stroke-width="12" stroke-linecap="round" stroke-linejoin="round"/><path d="M136 56H64C60.6863 56 58 58.6863 58 62V96V130C58 133.314 60.6863 136 64 136H136" stroke="black" stroke-width="12" stroke-linecap="round" stroke-linejoin="round"/>
</svg>
```

After (I'll add that to the PR)
```
<svg width="192" height="192" viewBox="0 0 192 192" fill="none" xmlns="http://www.w3.org/2000/svg"><rect x="30" y="30" width="132" height="132" rx="6" stroke="#000" stroke-width="12" stroke-linecap="round" stroke-linejoin="round"/><path d="M58 96h66m12-40H64a6 6 0 0 0-6 6v68a6 6 0 0 0 6 6h72" stroke="#000" stroke-width="12" stroke-linecap="round" stroke-linejoin="round"/></svg>
```

If it's not too much trouble, explain what good it does?

Comment (KTibow) [reply svgs/edupage.svg:1]: `rect` is easier to read and is minifies better (the rect version is 53% the size of the path versoin)

Comment (x9136) [reply svgs/edupage.svg:1]: @KTibow If you take the svgs from my comment, it's lighter without the `rect`.

Anyway, you think it's better if I disable `Shapes to (smaller) paths` before saving?

Screenshot 2023-10-28 at 16 55 51

Comment (KTibow) [reply svgs/edupage.svg:1]: > If you take the svgs from my comment, it's lighter without the rect.

Base SVG:
```
<rect x="30" y="30" width="132" height="132" rx="6"/>
```
Minified by it:
```
<path d="M162 156V36a6 6 0 00-6-6H36a6 6 0 00-6 6v120a6 6 0 006 6h120a6 6 0 006-6"/>
```
See above. Rect is more concise.

Comment (KTibow) [reply svgs/edupage.svg:1]: FYI `svgo` also minifies SVGs but only converts to a path if the path is smaller

Comment (x9136) [reply svgs/edupage.svg:1]: @KTibow Ok. The question is, which version should I add to the PR? "Before", "After" or a new one where I've turned off shape transformation in the path?

```
<svg width="192" height="192" viewBox="0 0 192 192" fill="none" xmlns="http://www.w3.org/2000/svg"><rect x="30" y="30" width="132" height="132" rx="6" stroke="#000" stroke-width="12" stroke-linecap="round" stroke-linejoin="round"/><path d="M58 96h66m12-40H64a6 6 0 0 0-6 6v68a6 6 0 0 0 6 6h72" stroke="#000" stroke-width="12" stroke-linecap="round" stroke-linejoin="round"/></svg>
```

Comment (KTibow) [reply svgs/edupage.svg:1]: Probably "After"

**Comment by x9136** [comment]
If there's any PR with optimization, it's better to deal with it first.

**Review by Chefski — APPROVED** [review]

**Pull request merged**